modules/dnn/src/layers/lrn_layer.cpp

```diff
@@ -412,7 +410,15 @@ class LRNLayerImpl CV_FINAL : public LRNLayer
             alphaSize *= (type == SPATIAL_NRM ? size*size : size);
 
         auto& ieInpNode = nodes[0].dynamicCast<InfEngineNgraphNode>()->node;
-        auto lrn = std::make_shared<ngraph::op::LRN>(ieInpNode, (double)alphaSize, (double)beta, (double)bias, (size_t)size);
+        std::vector<int64_t> axis;
+        if (type != SPATIAL_NRM) {
+            axis = {1};
+        } else {
+            axis.resize(ieInpNode->get_shape().size() - 2);
+            std::iota(axis.begin(), axis.end(), 2);
+        }
+        auto axes = std::make_shared<ngraph::op::Constant>(ngraph::element::i64, ngraph::Shape{axis.size()}, axis.data());
```

Comment:
> axis
> axes

Try to avoid mess with naming.
It make sense to add suffix/prefix for nGraph objects, like:
- `ngraph_axes`
- `axes_ng`